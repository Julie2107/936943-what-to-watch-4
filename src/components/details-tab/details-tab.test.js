import React from "react";
import renderer from "react-test-renderer";
import DetailsTab from "./details-tab.jsx";

const movie = {title: `Mindhunter`, poster: `http://placekitten.com/245/175`, cover: `img/bg-the-grand-budapest-hotel.jpg`, genre: `drama`, releaseYear: 2000, src: ``, rating: 5, ratingNumber: 100, ratingValue: ``, starring: [``, `1`, `3`], director: ``, reviews: []};

it(`Render details tab`, () => {
  const tree = renderer
    .create(<DetailsTab
      movie = {movie}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
