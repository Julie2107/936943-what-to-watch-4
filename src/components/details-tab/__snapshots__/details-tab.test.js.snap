// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render details tab 1`] = `
<div
  className="movie-card__text movie-card__row"
>
  <div
    className="movie-card__text-col"
  >
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Director
      </strong>
      <span
        className="movie-card__details-value"
      >
        
      </span>
    </p>
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Starring
      </strong>
      <span
        className="movie-card__details-value"
      >
        
         
        <br />
        1
         
        <br />
        3
         
        <br />
      </span>
    </p>
  </div>
  <div
    className="movie-card__text-col"
  >
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Run Time
      </strong>
      <span
        className="movie-card__details-value"
      >
        1h 39m
      </span>
    </p>
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Genre
      </strong>
      <span
        className="movie-card__details-value"
      >
        drama
      </span>
    </p>
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Released
      </strong>
      <span
        className="movie-card__details-value"
      >
        2000
      </span>
    </p>
  </div>
</div>
`;
